lmdeploy/model.py

```diff
@@ -1472,6 +1472,8 @@ def best_match_model(query: str) -> Optional[str]:
         from transformers import AutoTokenizer
         tokenizer_config = AutoTokenizer.from_pretrained(
             query, trust_remote_code=True)
+        if getattr('tokenizer_config', 'chat_template', None) is None:
```

Comment:
why is `tokenizer_config` a str instead of an object?

Reply:
I checked the original PR https://github.com/InternLM/lmdeploy/commit/2da44bffbcfd5644c62870fef4c128a5295e9260

But it seems the `best_match_model` should return a model name(`tokenizer_config.chat_template is None`) with the original pr, otherwise it will raise error.
https://github.com/AllentDan/lmdeploy/blob/67999be4f325578e064ca20d621d76ec83fe651b/lmdeploy/serve/async_engine.py#L56-L62